ClubManagement/ClubManagement.sln

```diff
@@ -5,6 +5,8 @@ VisualStudioVersion = 15.0.27004.2008
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ClubManagement", "ClubManagement\ClubManagement.csproj", "{3760470E-236C-497C-9958-C1FF8EF9B63C}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Event", "..\..\Event\Event\Event.csproj", "{6C5D50B9-572B-4902-B413-C4C90E29F147}"
```

Comment:
Where is the file. Please rename it to `DataModel` or something else.

Reply:
it is trash. Deleted